composer.json

```diff
@@ -5,7 +5,7 @@
     "minimum-stability": "dev",
     "description": "DKAN Open Data Catalog",
     "require": {
-        "drupal/admin_toolbar": "^2.3",
+        "drupal/admin_toolbar": "^3.2",
```

Comment:
I think this is now ^3 in the composer file, so maybe this change should be removed from the PR?